neph.yml

```diff
@@ -1,4 +1,4 @@
-main:
+ggmain:
```

Comment:
ah, I'm not familar with neph.yml. How can i fix that? I don't think i have modified this file.